packages/checkbox-group/stories/index.stories.mdx

```diff
@@ -0,0 +1,336 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { Required, Validator } from '@lion/validate';
+import '@lion/checkbox/lion-checkbox.js';
+
+import '../lion-checkbox-group.js';
+
+<Meta title="Forms/Checkbox Group" parameters={{ component: 'lion-checkbox-group' }} />
+
+# Checkbox Group
+
+`lion-checkbox-group` component enhances the functionality of the native `<input type="checkbox">` element.
+Its purpose is to provide a way for users to check **multiple** options amongst a set of choices, or to function as a single toggle.
+
+<Story name="Default">
+  {html`
```

Comment:
add `loadDefaultFeedbackMessages();` to show the default messages

Reply:
Done, works now :)